03/app.js

```diff
@@ -37,15 +37,29 @@ books.getAuthor = function(isbn) {
         return author;
     }
 
-    // brak informacji o autorze
+    // brak informacji o autorzes
     return false;
 }
 
 books.getTitle = function(isbn, lang) {
 
+    const title = this[isbn, lang]['title'];
```

Comment:
Tutaj jest niepoprawnie. Każde "zagnieżdżenie" to osobny nawias czyli `this[isbn]['title'][lang]`.
Kolejność też jest istotna, zobacz jaka jest struktura:
![image](https://github.com/devmentor-pl/practice-js-basics-objects/assets/55095257/8474a989-3449-4944-a154-c354a0859b6a)

Dodatkowo musimy sprawdzić czy dana wartość dynamiczna (zapisana w zmiennej) występuje czyli:
```
const isbn = this[isbn]
if(isbn) {
     const lang = isbn['title'][lang]
}

```